content/References/ZFSPrimer.md

```diff
@@ -1,179 +1,216 @@
-  The recommended number of disks per vdev is between 3 and 9.
-  With more disks, use multiple vdevs.
+Using more than 12 disks per vdev is not recommended. 
+The recommended number of disks per vdev is between 3 and 9. 
+With more disks, use multiple vdevs.
 
-* Some older ZFS documentation recommends that a certain number of disks is needed for each type of RAIDZ in order to achieve optimal performance.
-  On systems using LZ4 compression, which is the default for FreeNAS 9.2.1 and later, this is no longer true.
+Some older ZFS documentation recommends using a certain number of disks for each type of RAIDZ in order to achieve optimal performance.
+On systems using LZ4 compression, which is the default for FreeNAS 9.2.1 and later, this is no longer true.
 
 These resources can also help determine the RAID configuration best suited to the specific storage requirements:
 
 * [Getting the Most out of ZFS Pools](https://forums.freenas.org/index.php?threads/getting-the-most-out-of-zfs-pools.16/)
 
 * [A Closer Look at ZFS, Vdevs and Performance](https://constantin.glez.de/2010/06/04/a-closer-look-zfs-vdevs-and-performance/)
 
-> RAID AND DISK REDUNDANCY ARE NOT A SUBSTITUTE FOR A RELIABLE BACKUP STRATEGY.
-> BAD THINGS HAPPEN AND A GOOD BACKUP STRATEGY IS STILL REQUIRED TO PROTECT VALUABLE DATA.
-> Configuring TrueNAS with Periodic Snapshots and automated Replication is strongly recommended as part of a ZFS snapshot backup strategy.
+{{< hint warning >}}
+RAID and disk redundancy are not a substitute for a reliable backup strategy.
+Bad things happen and a good backup strategy is still required to protect valuable data.
+Configuring TrueNAS with periodic snapshots and automated replication is strongly recommended as part of a ZFS snapshot backup strategy.
+{{< /hint >}}
+
+### ZFS Device Management
 
-**ZFS manages devices**.
-When an individual drive in a mirror or RAIDZ fails and is replaced by the user, ZFS adds the replacement
-device to the vdev and copies redundant data to it in a process called *resilvering*.
-Hardware RAID controllers usually have no way of knowing which blocks were in use and must copy every block to the new device.
-ZFS only copies blocks that are in use, reducing the time it takes to rebuild the vdev.
-Resilvering is also interruptable.
-After an interruption, resilvering resumes where it left off rather than starting from the beginning.
+ZFS manage devices. 
+When an individual drive in a mirror or RAIDZ fails and is replaced by the user, ZFS adds the replacement device to the vdev and copies redundant data to it in a process called *resilvering*. 
+Hardware RAID controllers usually have no way of knowing which blocks are in use and must copy every block to the new device. 
+ZFS only copies blocks that are in use, reducing the time it takes to rebuild the vdev. 
+Resilvering is also interruptable. 
+After an interruption, resilvering resumes where it left off rather than starting from the beginning. 
```

Comment:
```suggestion
ZFS manages devices. 
When a user replaces a failed drive in a mirror or RAIDZ, ZFS adds the replacement device to the vdev and copies redundant data to it in a process called *resilvering*. 
Hardware RAID controllers usually do not know which blocks are in use, so they copy every block to the new device. 
ZFS only copies blocks in use, reducing the time it takes to rebuild the vdev. 
Resilvering is also interruptible. 
After an interruption, resilvering resumes where it left off rather than starting from the beginning. 
```